filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -385,6 +385,10 @@ filebeat.inputs:
   # Note: Potential data loss. Make sure to read and understand the docs for this option.
   #close.reader.after_interval: 0
 
+  # Collect detailed metrics about each opened file. Information include: size of the file, offset
+  # last time an event was published, state ID, etc.
+  #detailed_metrics: false
```

Comment:
Maybe we want to make this feature experimental? At some point we might want to remove the setting, but keep the feature.

Reply:
If we are keeping the feature, why make it experimental? Maybe instead of removing the option later, we should rather remove it now. As I understood the `/datastream` endpoint is only available if HTTP endpoint is enabled, right? So basically it is already confiurable.

Reply:
I'm ok with removing the setting.